### PR TITLE
Dropdown changes

### PR DESCRIPTION
**Pull request by SebastianGode**

### CODE REVIEW

File: otcdocstheme/theme/otcdocs/static/css/otcdocstheme.css [hunk 1]
```diff
@@ -222,8 +222,8 @@ body {
 
 @media screen and (min-width: 768px) {
   #navbarNav .navbar-nav > li > a.nav-link {
-    color: var(--dt-color-grey-10);
-    font-weight: 300;
+    color: var(--dt-color-white);
```
Comment (gtema) [inline otcdocstheme/theme/otcdocs/static/css/otcdocstheme.css:225]: Agree with setting font back to white, cause contrast is not high enough

File: otcdocstheme/theme/otcdocs/static/css/otcdocstheme.css [hunk 2]
```diff
@@ -879,3 +879,8 @@ pre .cl {
 .btn-info {
   background-color: var(--dt-color-teal-30);
 }
+
+/* Open Dropdown on Hover */
+.dropdown:hover .dropdown-menu {
```
Comment (gtema) [inline otcdocstheme/theme/otcdocs/static/css/otcdocstheme.css:884]: Here I am not really sure - i.e. K8 is not doing that and I often really struggle with auto-opening things on mobile - this is killing UX and not improving it

Comment (SebastianGode) [reply otcdocstheme/theme/otcdocs/static/css/otcdocstheme.css:884]: I looked at the OpenStack Docu
https://docs.openstack.org/xena/

However I think that is a personal design decision. I would like a vote for that and depending on that we let it on click-only or make it on hover

**Review by tischrei — APPROVED** [review]

**Review by gtema — CHANGES_REQUESTED** [review]
If accepted please ensure in the mobile view this does not land between navbars

**Comment by gtema** [comment]
sorry, discard comment - wrong PR

**Review by gtema — APPROVED** [review]

**Pull request merged**